hubblestack/extmods/audit/stat.py

```diff
@@ -0,0 +1,202 @@
+# -*- encoding: utf-8 -*-
+"""
+Nova module for running stat command
+
+Example for a check implementation in a profile:
+check_unique_id:
+  description: 'stat check'
+  tag: 'ADOBE-01'
+  implementations:
+    - filter:
+        grains: 'G@osfinger:CentOS*Linux-7'
+
+      hubble_version: '>3 AND <7 AND <8'
+
+      module: stat
+
+      checks:
+        - path: /etc/ssh/ssh_config1
+          gid: 0
+          group: root
+          mode: 644
+          uid: 0
+          user: root
+          allow_more_strict: true
+
+It is checking for file on the given path and matches the permissions on it with given params.
+Mandatory parameters:
+    path - file path
+    gid - group id
+    group - group name
+    mode - file mode
+    uid - user id
+    user - user name
```

Comment:
Maybe we should allow only some of the parameters? Would we ever only want to enforce some of these options in a check?